e2e/coverage-transform-instrumented/package.json

```diff
@@ -4,6 +4,11 @@
     "transform": {
       "^.+\\.(js)$": "<rootDir>/preprocessor.js"
     },
+    "transformIgnorePatterns": [
+      "jest-jasmine2",
+      "jest-each",
+      "jest-circus"
+    ],
```

Comment:
These will have to be ignored manually as a relative path to the package is used rather than node_modules.
Seems like the number of ignore paths might increase going forward, one idea I had was to add a pattern like `jest-`, but that doesn't seem to work in all cases